docs/uml-class-diagrams/database/production/DatabaseGisManagerImplementation/DatabaseGisManagerImplementation.puml~

```diff
@@ -0,0 +1,14 @@
+@startuml DatabaseGisManagerImplementation
```

Comment:
This file should be deleted --> it is a temporary file and should not be tracked on GitHub.